Purchases/Networking/Operations/Handling/SubscriberAttributesMarshaller.swift

```diff
@@ -13,9 +13,10 @@
 
 import Foundation
 
-class SubscriberAttributesMarshaller {
+enum SubscriberAttributesMarshaller {
 
-    func map(subscriberAttributes: SubscriberAttributeDict) -> [String: [String: Any]] {
+    // fixme: make `SubscriberAttributeDict` `Encodable` instead
+    static func map(subscriberAttributes: SubscriberAttributeDict) -> [String: [String: Any]] {
```

Comment:
is there a follow-up PR for this?

Reply:
Oh no thanks for reminding me about this.